NSrtm.Core/Pgm/PgmDataDescription.cs

```diff
@@ -0,0 +1,186 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace NSrtm.Core.Pgm
+{
+    public struct PgmDataDescription : IEquatable<PgmDataDescription>
+    {
+        private readonly double _offset;
+        private readonly double _scale;
+        private readonly int _originLat;
+        private readonly int _originLon;
+        private readonly int _gridGraphWidthPoints;
+        private readonly int _gridGraphHeightPoints;
+        private readonly int _maxValue;
+        private readonly int _preambleLength;
+        private readonly int _numberOfPoints;
+        private readonly double _latitudeIncrement;
+        private readonly double _longitudeIncrement;
+
+        public PgmDataDescription(
+            double offset,
+            double scale,
+            int originLat,
+            int originLon,
+            int gridGraphWidthPoints,
+            int gridGraphHeightPoints,
+            int maxValue,
+            int preambleLength)
+        {
+            _offset = offset;
+            _scale = scale;
+            _originLat = originLat;
+            _originLon = originLon;
+            _gridGraphHeightPoints = gridGraphHeightPoints;
+            _gridGraphWidthPoints = gridGraphWidthPoints;
+            _maxValue = maxValue;
+            _preambleLength = preambleLength;
+            _numberOfPoints = _gridGraphHeightPoints * _gridGraphWidthPoints;
+            _longitudeIncrement = _gridGraphWidthPoints / 360.0;
+            _latitudeIncrement = (_gridGraphHeightPoints - 1) / 180.0;
+        }
+
+        public override bool Equals(object obj)
+        {
+            if (ReferenceEquals(null, obj)) return false;
+            return obj is PgmDataDescription && Equals((PgmDataDescription)obj);
+        }
+
+        public override int GetHashCode()
```

Comment:
By mistake I attached an unnecessary layers with CA warnings. I thought it was our rules.